pkg/server/handler/me.go

```diff
@@ -26,6 +27,7 @@ type MeHandler struct {
 	Authenticator router.Processor `preprocessor:"authenticator"`
 	DBConn        router.Processor `preprocessor:"dbconn"`
 	InjectUser    router.Processor `preprocessor:"inject_user"`
+	InjectDB      router.Processor `preprocessor:"inject_db"`
```

Comment:
Don’t use the `InjectDB`. Suppose the client sends `_database_id = "_private"` to the `me` endpoint your code will attempt to get user record from the private DB, which doesn’t exist.